elifetools/parseJATS.py

```diff
@@ -2471,6 +2487,13 @@ def body_block_content(tag, html_flag=True, base_url=None):
             caption_content, supplementary_material_tags = body_block_caption_render(caption_tags, base_url=base_url)
         body_block_title_label_caption(asset_tag_content, title_value, label_value, caption_content, set_caption=True)
 
+        # license or attribution
+        attributions = body_block_attribution(tag)
+        if attributions:
+            asset_tag_content["attribution"] = []
```

Comment:
I assume it's fine for the `attribution` key of the dictionary to be missing if there is no content

Reply:
There are already many blocks having no ``attribution``, indicating the schema allows the attribute to be optional. When ``attribution`` is present, it must be a list / array, and whether it can be a blank list I'm not sure. Here the attributed is added only if there are attributions to set.